src/main/resources/public/js/getAndAddContracts.js

```diff
@@ -157,13 +157,17 @@ var projectContractsCount = -1;
                 + "</a>  ";
             if (invoice.paymentTime == "null" && invoice.transactionId == "null") {
                 status = "Active";
-                if (parseFloat(invoice.totalAmount) > 0.0) {
+                var totalAmount = parseFloat(invoice.totalAmount.substring(0, invoice.totalAmount.length - 1))
```

Comment:
I think I have a bug here though, total amount has comma as decimal separator and messes the parseFloat.